Detectors/TPC/workflow/src/ChunkedDigitPublisher.cxx

```diff
@@ -199,6 +199,7 @@ void publishMergedTimeframes(std::vector<int> const& lanes, std::vector<int> con
     //data definitions
     using DigitsType = std::vector<o2::tpc::Digit>;
     using LabelType = o2::dataformats::MCTruthContainer<o2::MCCompLabel>;
+#pragma omp critical
```

Comment:
It probably then needs to be applied to every function invocation, not just this one (and then it becomes ugly).
Also now the lock will be on a  much larger section of code than before. So, in the end, we will probably need to convert the lambda into a regular template function.